structs_marshal.go

```diff
@@ -192,6 +192,9 @@ func (c *CallMsg) MarshalJSON() ([]byte, error) {
 	if c.Value != nil {
 		o.Set("value", a.NewString(fmt.Sprintf("0x%x", c.Value)))
 	}
+	if c.Gas != nil {
+		o.Set("gas", a.NewString(fmt.Sprintf("0x%x", c.Gas)))
```

Comment:
Is it `gas` or `gasLimit`?

Reply:
It's `gas`, as by the [spec](https://eth.wiki/json-rpc/API#eth_call)